<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title> Hot takes: interesting papers from Feb —  &raquo;  Notes</title>
<meta name="description" content="">
<meta name="keywords" content="">
<link rel="canonical" href="/2016-03/papers.html">
<script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>


          
  
  
  

<!-- Twitter Cards -->
<meta name="twitter:card" content="summary" />
<meta name="twitter:title" content="Hot takes: interesting papers from Feb" />
<meta name="twitter:description" content="" />
<meta name="twitter:image" content="" />

<!-- Google plus -->
<meta name="author" content="">
<link rel="author" href="">

<!-- Open Graph -->
<meta property="og:locale" content="">
<meta property="og:type" content="article">
<meta property="og:title" content="Hot takes: interesting papers from Feb">
<meta property="og:description" content="">
<meta property="og:url" content="/2016-03/papers.html">
<meta property="og:site_name" content="Notes">

        <link href='http://fonts.googleapis.com/css?family=Inconsolata:400,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/normalize-css/normalize.css">
<link rel="stylesheet" href="http://sashagusev.github.io/css/main.css">
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/highlight/styles/hopscotch.css">
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/font-awesome/css/font-awesome.css">

    </head>

    <body>
        <div class="wrapper">
            <header class="header">
    <div class="navigation">
        <a href="/" class="logo">Notes</a>
        <br />
		My name is Sasha Gusev. <br />
		I study the genetics of complex traits.<br />
		I work <a href="http://gusevlab.org/">here</a> on <a href="https://scholar.google.com/citations?user=mRCDpEIAAAAJ&hl=en">these</a>.
		<br />
		Reach me at sgusev@gmail.com
    </div>
</header>


            <h1 class="page-title post-title">
    <div class="page-title__text post-title__text">Hot takes: interesting papers from Feb</div>
    <div class="page-title__subtitle post-title__subtitle"></div>
</h1>

<div class="content">
    <p><em>Intriguing papers that were published in the previous month, with highlights.</em></p>

<h2 id="functional-annotation">Functional annotation</h2>

<p><strong><a href="http://journals.plos.org/plosgenetics/article?id=10.1371%2Fjournal.pgen.1005875">Which Genetics Variants in DNase-Seq Footprints Are More Likely to Alter Binding?, Moyerbrailean et al. PLOS Genet</a></strong></p>

<p>“Here, we integrated DNaseI footprinting data with sequence-based transcription factor (TF) motif models to predict the impact of a genetic variant on TF binding across 153 tissues and 1,372 TF motifs … As an example, the enrichment for LDL level-associated SNPs is 9.1-fold higher among SNPs predicted to affect HNF4 binding sites than in a background model already including tissue-specific annotation.”</p>

<p><strong><a href="http://www.nature.com/ng/journal/v48/n3/full/ng.3507.html">Weighting sequence variants based on their annotation increases power of whole-genome association studies, Sveinbjornsson et al. Nat Genet</a></strong></p>

<p>“Functional annotations can be used to improve power to detect associations. To increase power by incorporating the information that these annotations hold, we suggest a weighting scheme based on the observed enrichment using a weighted Bonferroni correction similar to that suggested by Roeder et al. for linkage analysis … we note that the number of significant signals using the enrichments as weights (n = 166) was not far from being optimal (n = 185), and this approach had more power than the standard Bonferroni method (n = 146) to detect associations … The commonly accepted P = 5 × 10^−8 threshold is outdated and will not be applicable in future GWAS.”</p>

<p><strong><a href="http://www.nature.com/ng/journal/v48/n2/full/ng.3477.html">A spectral approach integrating functional genomic annotations for coding and noncoding variants, Ionita-Laza et al. Nat Genet</a></strong></p>

<p>“Here we develop an unsupervised approach to integrate these different annotations into one measure of functional importance that, unlike most existing methods, is not based on any labeled training data. We show that the resulting meta-score has better discriminatory ability using disease-associated and putatively benign variants from published studies (in both coding and noncoding regions) than the recently proposed CADD score.”</p>

<h2 id="gene-expression">Gene expression</h2>

<p><strong><a href="http://www.cell.com/ajhg/abstract/S0002-9297(16)00004-5">A Burden of Rare Variants Associated with Extremes of Gene Expression in Human Peripheral Blood, Zhao et al. AJHG</a></strong></p>

<p>“After sequencing 2-kb promoter regions of 472 genes in 410 healthy adults, we performed a quadratic regression of rare variant count on bins of peripheral blood transcript abundance from microarrays. The overall burden test results are consistent with rare and private regulatory variants driving high or low transcription at specific loci, potentially contributing to disease.”</p>

<h2 id="gwas">GWAS</h2>

<p><strong><a href="http://journals.plos.org/plosgenetics/article?id=10.1371%2Fjournal.pgen.1005765">G = E: What GWAS Can Tell Us about the Environment, Gage et al. PLOS Genet</a></strong></p>

<p>“as large, richly phenotyped cohort studies (e.g., UK Biobank) emerge, it will become possible to identify modifiable exposures from genetic data and to dissect those pathways within the same cohort … A failure to appreciate this point will hamper our ability to translate the results of GWAS into health benefits, by focusing attention on possible biological pathways when, in fact, the target for intervention could be a modifiable environmental or behavioural exposure.”</p>

<p><strong><a href="http://www.nature.com/nature/journal/v530/n7589/full/nature16549.html">Schizophrenia risk from complex variation of complement component 4, Sekar et al. Nature</a></strong></p>

<p>“Schizophrenia’s strongest genetic association at a population level involves variation in the major histocompatibility complex (MHC) locus, but the genes and molecular mechanisms accounting for this have been challenging to identify. Here we show that this association arises in part from many structurally diverse alleles of the complement component 4 (C4) genes … These results implicate excessive complement activity in the development of schizophrenia and may help explain the reduced numbers of synapses in the brains of individuals with schizophrenia.”</p>

<p><strong><a href="http://www.cell.com/ajhg/fulltext/S0002-9297(15)00514-5">A Robust Example of Collider Bias in a Genetic Association Study, Day et al. AJHG</a></strong></p>

<p>“In summary, we have demonstrated that adjusting for causally associated covariates can create apparently highly robust, but actually biologically spurious, associations. The extent of this collider bias is almost perfectly inversely related to the strength of the exposure-collider association. Consideration of causal inference modeling and unadjusted test statistics is therefore of great importance in the design and interpretation of genetic (and non-genetic1) association studies.”</p>

<h2 id="popgen">Popgen</h2>

<p><strong><a href="http://www.nature.com/nature/journal/v530/n7591/full/nature16544.html">Ancient gene flow from early modern humans into Eastern Neanderthals, Kuhlwilm et al. Nature</a></strong></p>

<p>“We conclude that in addition to later interbreeding events, the ancestors of Neanderthals from the Altai Mountains and early modern humans met and interbred, possibly in the Near East, many thousands of years earlier than previously thought.”</p>

<p><strong><a href="http://science.sciencemag.org/content/351/6275/aaf3945">Erratum for the Report “Ancient Ethiopian genome reveals extensive Eurasian admixture in Eastern Africa”, Gallego et al. Science</a></strong></p>

<p>“A script necessary to convert the input produced by samtools v0.1.19 to be compatible with PLINK was not run when merging the ancient genome, Mota, with the contemporary populations SNP panel, leading to homozygote positions to the human reference genome being dropped as missing data (the analysis of admixture with Neandertals and Denisovans was not affected) … the geographic extent of the genetic impact of this migration was overestimated: The Western Eurasian backflow mostly affected East Africa and only a few Sub-Saharan populations; the Yoruba and Mbuti do not show higher levels of Western Eurasian ancestry compared to Mota.”</p>

<p><strong><a href="http://www.cell.com/ajhg/fulltext/S0002-9297(16)00011-2">The Kalash Genetic Isolate? The Evidence for Recent Admixture, Hellenthal et al. AJHG</a></strong></p>

<p>“These observations indicate that, contrary to the claim of Ayub et al. that the ancestors of the Kalash have been isolated from the ancestors of other extant populations for over 8,000 years, there is in fact strong evidence that they have not been isolated over this time frame.”</p>

<p><strong><a href="http://genome.cshlp.org/content/26/3/279.full">Whole-genome sequence analyses of Western Central African Pygmy hunter-gatherers reveal a complex demographic history and identify candidate genes under positive natural selection, Hsieh et al. Gen Res</a></strong></p>

<p>“we sequenced the genomes of four Biaka Pygmies … we fit models using the joint allele frequency spectrum … Our two best-fit models both suggest ancient divergence between the ancestors of the farmers and Pygmies, 90,000 or 150,000 yr ago.”</p>

<p><strong><a href="http://genome.cshlp.org/content/26/3/291.full">Model-based analyses of whole-genome data reveal a complex evolutionary history involving archaic introgression in Central African Pygmies, Hsieh et al. Gen Res</a></strong></p>

<p>“Our inference method rejects the hypothesis that the ancestors of [anatomically modern humans] were genetically isolated in Africa, thus providing the first whole genome-level evidence of African archaic admixture. Our inferences also suggest a complex human evolutionary history in Africa, which involves at least a single admixture event from an unknown archaic population into the ancestors of AMH, likely within the last 30,000 yr.”</p>

</div>

<div class="about">
    <div class="about__devider">*****</div>
    <div class="about__text">
        Written by <strong>  Sasha Gusev </strong>
        on <strong>01 March 2016</strong>
    </div>
</div>


<div id="disqus_thread"></div>
<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES * * */
    var disqus_shortname = 'sashagusev';

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>



        </div>

        <script src="http://sashagusev.github.io/assets/vendor/highlight/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
        
    </body>
</html>