<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title> Hot takes: interesting papers from Apr/May —  &raquo;  Sasha Gusev</title>
<meta name="description" content="scientist">
<meta name="keywords" content="">
<link rel="canonical" href="/2016-06/papers.html">
<script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>


          
  
  
  

<!-- Twitter Cards -->
<meta name="twitter:card" content="summary" />
<meta name="twitter:title" content="Hot takes: interesting papers from Apr/May" />
<meta name="twitter:description" content="scientist" />
<meta name="twitter:image" content="" />

<!-- Google plus -->
<meta name="author" content="">
<link rel="author" href="">

<!-- Open Graph -->
<meta property="og:locale" content="">
<meta property="og:type" content="article">
<meta property="og:title" content="Hot takes: interesting papers from Apr/May">
<meta property="og:description" content="scientist">
<meta property="og:url" content="/2016-06/papers.html">
<meta property="og:site_name" content="Sasha Gusev">

        <link href='http://fonts.googleapis.com/css?family=Inconsolata:400,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/normalize-css/normalize.css">
<link rel="stylesheet" href="http://sashagusev.github.io/css/main.css">
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/highlight/styles/hopscotch.css">
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/font-awesome/css/font-awesome.css">

    </head>

    <body>
        <div class="wrapper">
            <header class="header">
    <div class="navigation">
        <a href="/" class="logo">Sasha Gusev</a>

        <ul class="menu">
            <li class="menu__entry"><a href="http://sashagusev.github.io/about">About</a></li>
            <li class="menu__entry"><a href="http://sashagusev.github.io/">Blog/Notes</a></li>
            <li class="menu__entry"><a href="http://sashagusev.github.io/links">Links</a></li>
        </ul>
    </div>

    <ul class="social-links">
        
            <a href="https://github.com/sashagusev" class="social-links__entry" target="_blank">
                <i class="fa fa-github"></i>
            </a>
        

        
    </ul>
</header>


            <h1 class="page-title post-title">
    <div class="page-title__text post-title__text">Hot takes: interesting papers from Apr/May</div>
    <div class="page-title__subtitle post-title__subtitle"></div>
</h1>

<div class="content">
    <p><em>Better late than never</em></p>

<h2 id="gwascausal-mechanisms">GWAS/causal mechanisms</h2>

<p><strong><a href="http://www.nature.com/nature/journal/v533/n7604/full/nature17671.html">Genome-wide association study identifies 74 loci associated with educational attainment, Okbay et al. Nature</a></strong></p>

<p><em>The Supplemental Materials to this paper contain the most thorough and clear description of cutting-edge GWAS analyses I have read of late</em></p>

<p><strong><a href="http://www.nature.com/nature/journal/v533/n7601/full/nature17939.html">Parkinson-associated risk variant in distal enhancer of α-synuclein modulates target gene expression, Soldner et al. Nature</a></strong></p>

<p>“Here we describe an alternative experimental approach to identify functional risk variants based on three recent innovations in genetics and molecular biology: (i) the prioritization of GWAS-identified risk variants in regulatory elements such as distal enhancers annotated based on genome-scale epigenetic data; (ii) the generation of genetically controlled isogenic pluripotent stem cell lines in which specific disease-associated genetic variants are the sole modified experimental variable using efficient gene-editing technologies such as the CRISPR/Cas9 system; and (iii) the analysis of cis-acting effects of candidate variants on allele-specific gene expression through deletion or exchange of disease-associated regulatory elements.”</p>

<p><strong><a href="http://www.nature.com/ng/journal/vaop/ncurrent/full/ng.3570.html">Detection and interpretation of shared genetic influences on 42 human traits, Pickrell et al. Nat Genet</a></strong></p>

<p>“It is also striking to note how many genetic variants influence multiple traits but without a consistent correlation in effect sizes … Another possibility is that a given genetic variant often influences the function of multiple cell types through separate molecular pathways or that the effects of a variant on two related phenotypes vary according to an individual’s environmental exposures.”</p>

<h2 id="regulatory-mechanisms">Regulatory mechanisms</h2>

<p><strong><a href="http://www.nature.com/nmeth/journal/v13/n4/abs/nmeth.3799.html">Tissue-specific regulatory circuits reveal variable modular perturbations across complex diseases, Marbach et al. Nat Methods</a></strong></p>

<p>“For most traits, evidence of increased connectivity between perturbed genes extended to variants that did not pass the genome-wide significance threshold, indicating that regulatory network information will be useful for prioritizing candidate variants.”</p>

<p><strong><a href="http://www.cell.com/cell/abstract/S0092-8674(16)30339-7">Pooled ChIP-Seq Links Variation in Transcription Factor Binding to Complex Disease Risk, Tehranchi et al. Cell</a></strong></p>

<p>“Altogether, 3,601 of our bQTLs have been previously implicated by GWAS, either directly or indirectly via LD (r2 &gt; 0.8 in YRI). These represent 2,282 different disease-associated variants, 995 (44%) of which are associated with bQTLs for multiple factors. Interestingly, this is 8.0-fold higher than the overall fraction of bQTLs associated with multiple factors, suggesting that variants affecting multiple TFs are more likely to impact phenotypes … intersecting GWAS loci with all binding sites of a TF may yield misleading results because these overlaps are dominated by SNPs with no effect on TF binding.”</p>

<p><strong><a href="http://www.pnas.org/content/113/23/6508.abstract">Syntax compensates for poor binding sites to encode tissue specificity of developmental enhancers, Farley et al. PNAS</a></strong></p>

<p>“Surprisingly, enhancers with low-affinity binding sites can mediate robust tissue specific patterns of gene expression when they are organized with optimal syntax. Such enhancers may be a vastly underappreciated feature of the regulatory genome.”</p>

<p><strong><a href="http://genome.cshlp.org/content/early/2016/05/03/gr.200535.115.abstract">Basset: Learning the regulatory code of the accessible genome with deep convolutional neural networks, Kelley et al. Gen Res</a></strong></p>

<p>“We introduce an open source package Basset to apply CNNs [deep convolutional neural networks] to learn the functional activity of DNA sequences from genomics data. We trained Basset on a compendium of accessible genomic sites mapped in 164 cell types by DNase-seq and demonstrate greater predictive accuracy than previous methods. Basset predictions for the change in accessibility between variant alleles were far greater for GWAS SNPs that are likely to be causal relative to nearby SNPs in linkage disequilibrium with them.”</p>

<p><strong><a href="http://www.nature.com/ng/journal/v48/n5/abs/ng.3539.html">Enhancer–promoter interactions are encoded by complex genomic signatures on looping chromatin, Whalen et al. Nat Genet</a></strong></p>

<p>“The resulting models accurately predict individual enhancer–promoter interactions across multiple cell lines with a false discovery rate up to 15 times smaller than that obtained using the closest gene …  Most of this signature is not proximal to the enhancers and promoters but instead decorates the looping DNA.”</p>

<h2 id="gene-expression">Gene expression</h2>

<p><strong><a href="http://science.sciencemag.org/content/352/6285/600.long">RNA splicing is a primary link between genetic variation and disease, Li et al. Science</a></strong></p>

<p>“About ~65% of expression quantitative trait loci (eQTLs) have primary effects on chromatin, whereas the remaining eQTLs are enriched in transcribed regions … splicing QTLs are major contributors to complex traits, roughly on a par with variants that affect gene expression levels.”</p>

<p><strong><a href="http://www.cell.com/ajhg/abstract/S0002-9297(16)00071-9">Imputing Gene Expression in Uncollected Tissues Within and Beyond GTEx, Wang et al. AJHG</a></strong></p>

<p>“By analyzing data from nine selected tissue types in the GTEx pilot project, we demonstrated that harnessing expression quantitative trait loci (eQTLs) and tissue-tissue expression-level correlations can aid imputation of transcriptome data from uncollected GTEx tissues. More importantly, we showed that by using GTEx data as a reference, one can impute expression levels in inaccessible tissues in non-GTEx expression studies.”</p>

</div>

<div class="about">
    <div class="about__devider">*****</div>
    <div class="about__text">
        Written by <strong>  Sasha Gusev </strong>
        on <strong>17 June 2016</strong>
    </div>
</div>


<div id="disqus_thread"></div>
<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES * * */
    var disqus_shortname = 'sashagusev';

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>



        </div>

        <script src="http://sashagusev.github.io/assets/vendor/highlight/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
        
    </body>
</html>