<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title> Hot takes: interesting papers from January —  &raquo;  Sasha Gusev</title>
<meta name="description" content="scientist">
<meta name="keywords" content="">
<link rel="canonical" href="/2016-02/papers.html">
<script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>


          
  
  
  

<!-- Twitter Cards -->
<meta name="twitter:card" content="summary" />
<meta name="twitter:title" content="Hot takes: interesting papers from January" />
<meta name="twitter:description" content="scientist" />
<meta name="twitter:image" content="" />

<!-- Google plus -->
<meta name="author" content="">
<link rel="author" href="">

<!-- Open Graph -->
<meta property="og:locale" content="">
<meta property="og:type" content="article">
<meta property="og:title" content="Hot takes: interesting papers from January">
<meta property="og:description" content="scientist">
<meta property="og:url" content="/2016-02/papers.html">
<meta property="og:site_name" content="Sasha Gusev">

        <link href='http://fonts.googleapis.com/css?family=Inconsolata:400,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/normalize-css/normalize.css">
<link rel="stylesheet" href="http://sashagusev.github.io/css/main.css">
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/highlight/styles/hopscotch.css">
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/font-awesome/css/font-awesome.css">

    </head>

    <body>
        <div class="wrapper">
            <header class="header">
    <div class="navigation">
        <a href="/" class="logo">Sasha Gusev</a>

        <ul class="menu">
            <li class="menu__entry"><a href="http://sashagusev.github.io/about">About</a></li>
            <li class="menu__entry"><a href="http://sashagusev.github.io/">Blog/Notes</a></li>
            <li class="menu__entry"><a href="http://sashagusev.github.io/links">Links</a></li>
        </ul>
    </div>

    <ul class="social-links">
        
            <a href="https://github.com/sashagusev" class="social-links__entry" target="_blank">
                <i class="fa fa-github"></i>
            </a>
        

        
    </ul>
</header>


            <h1 class="page-title post-title">
    <div class="page-title__text post-title__text">Hot takes: interesting papers from January</div>
    <div class="page-title__subtitle post-title__subtitle"></div>
</h1>

<div class="content">
    <p><em>Intriguing papers that were published in the previous month, with highlights.</em></p>

<h2 id="heritability">Heritability</h2>

<p><strong><a href="http://www.nature.com/ng/journal/v48/n1/full/ng.3446.html">The contribution of rare variation to prostate cancer heritability, Mancuso et al. Nat Genet</a></strong></p>

<p>“Our finding that 42% (95% confidence interval = 21–63%) of the genetic risk for prostate cancer is due to variants in the MAF range of 0.1–1% is striking, given that only a couple percent of neutral varia- tion is due to SNPs in this frequency range.”</p>

<p><strong><a href="http://www.nature.com/ng/journal/v48/n1/abs/ng.3461.html">Abundant contribution of short tandem repeats to gene expression variation in humans, Gymrek et al. Nat Genet</a></strong></p>

<p>“We used variance partitioning to disentangle the contribution of eSTRs from that of linked SNPs and indels and found that eSTRs contribute 10–15% of the cis heritability [of expression] mediated by all common variants.”</p>

<p>“We hypothesize that there are more eSTRs to find in the genome…”</p>

<h2 id="population-genetics">Population genetics</h2>

<p><strong><a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00485-1">Genomic Signatures of Selective Pressures and Introgression from Archaic Hominids at Human Innate Immunity Genes, Deschamps et al. AJHG</a></strong></p>

<p>“Using full-genome sequence variation from the 1000 Genomes Project, we first show that innate immunity genes have globally evolved under stronger purifying selection than the remainder of protein-coding genes … Finally, we show that innate immunity genes present higher Neandertal intro- gression than the remainder of the coding genome.”</p>

<p><strong><a href="http://www.nature.com/ng/journal/v48/n1/full/ng.3464.html">Visualizing spatial population structure with estimated effective migration surfaces, Petkova et al. Nat Genet</a></strong></p>

<p>“EEMS is a new method for analyzing population structure from geo-referenced genetic samples. EEMS produces an intuitive visual representation of spatial patterns in genetic variation and highlights regions of higher-than-average and lower-than-average historical gene flow.”</p>

<p>“Distance matrices based on rare SNPs could also provide insights into more recent dispersal history…”</p>

<p><strong><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005703">A Spatial Framework for Understanding Population Structure and Admixture, Bradburd et al. PLOS Gen</a></strong></p>

<p>“We use genome-wide polymorphism data to build “geo- genetic maps,” which, when applied to stationary populations, produces a map of the geo- graphic positions of the populations, but with distances distorted to reflect historical rates of gene flow.”</p>

<p>“Additionally, although we have focused on the covariance among alleles at the same locus, linkage disequilibrium (covariance of alleles among loci) holds rich information about the timing and source of admixture events as well as information about isolation by distance.”</p>

<p>“The inclusion of ancient DNA samples in the analyzed sample offers a ￼way to get better representation of the ancestral populations from which the ancestors of modern samples received their admixture.”</p>

<h2 id="gene-expression">Gene Expression</h2>

<p><strong><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005793">Genetic Variation, Not Cell Type of Origin, Underlies the Majority of Identifiable Regulatory Differences in iPSCs, Burrows et al. PLOS Gen</a></strong></p>

<p>“We show that the cell type of origin only minimally affects gene expression levels and DNA methylation in iPSCs (induced pluripotent stem cells), and that genetic variation is the main driver of regulatory differences between iPSCs of different donors. Our findings suggest that studies using iPSCs should focus on additional individuals rather than clones from the same individual.”</p>

<h2 id="gwas">GWAS</h2>

<p><strong><a href="http://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1005803">Leveraging Genomic Annotations and Pleiotropic Enrichment for Improved Replication Rates in Schizophrenia GWAS, Wang et al. PLOS Gen</a></strong></p>

<p>“We have presented a novel algorithm, called CM3, which provides more accurate estimates of predicted replication probabilities for each SNP in a GWAS. Sorting SNPs based on predicted finite-sample replication probabilities incorporating auxiliary information, rather than by nominal p-values, yields a larger number of SNPs for a given replication threshold.”</p>

<p>“An important utility of the CM3 method may be selection of a greater proportion of relevant SNPs for gene set enrichment and biological pathway analyses…”</p>

<h2 id="big-data">Big Data</h2>

<p><strong><a href="http://www.cell.com/ajhg/references/S0002-9297(15)00491-7">Genotype Imputation with Millions of Reference Samples, Browning &amp; Browning AJHG</a></strong></p>

<p>“We demonstrate that Beagle v.4.1 scales to much larger reference panels [than IMPUTE or Minimac] by performing imputation from a simulated reference panel having 5 million samples”</p>

<p>“When there are millions of reference samples, use of a binary reference file can reduce wall clock computation time by &gt;80%.”</p>

<p>“With a reference panel containing 200,000 simulated European individuals, we find that markers with at least nine copies of the minor allele in the reference panel can be imputed with high accuracy (r2 &gt; 0.8) in target samples that have been genotyped with a 1M SNP array.”</p>

</div>

<div class="about">
    <div class="about__devider">*****</div>
    <div class="about__text">
        Written by <strong>  Sasha Gusev </strong>
        on <strong>04 February 2016</strong>
    </div>
</div>


<div id="disqus_thread"></div>
<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES * * */
    var disqus_shortname = 'sashagusev';

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>



        </div>

        <script src="http://sashagusev.github.io/assets/vendor/highlight/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
        
    </body>
</html>