<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title> Hot takes: interesting papers from June —  &raquo;  Notes</title>
<meta name="description" content="">
<meta name="keywords" content="">
<link rel="canonical" href="/2016-07/papers.html">
<script src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>


          
  
  
  

<!-- Twitter Cards -->
<meta name="twitter:card" content="summary" />
<meta name="twitter:title" content="Hot takes: interesting papers from June" />
<meta name="twitter:description" content="" />
<meta name="twitter:image" content="" />

<!-- Google plus -->
<meta name="author" content="">
<link rel="author" href="">

<!-- Open Graph -->
<meta property="og:locale" content="">
<meta property="og:type" content="article">
<meta property="og:title" content="Hot takes: interesting papers from June">
<meta property="og:description" content="">
<meta property="og:url" content="/2016-07/papers.html">
<meta property="og:site_name" content="Notes">

        <link href='http://fonts.googleapis.com/css?family=Inconsolata:400,700' rel='stylesheet' type='text/css'>
<link href='http://fonts.googleapis.com/css?family=Montserrat:400,700' rel='stylesheet' type='text/css'>
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/normalize-css/normalize.css">
<link rel="stylesheet" href="http://sashagusev.github.io/css/main.css">
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/highlight/styles/hopscotch.css">
<link rel="stylesheet" href="http://sashagusev.github.io/assets/vendor/font-awesome/css/font-awesome.css">

    </head>

    <body>
        <div class="wrapper">
            <header class="header">
    <div class="navigation">
        <a href="/" class="logo">Notes</a>
        <br />
		My name is Sasha Gusev. <br />
		I study the genetics of complex traits.<br />
		I work <a href="http://gusevlab.org/">here</a> on <a href="https://scholar.google.com/citations?user=mRCDpEIAAAAJ&hl=en">these</a>.
		<br />
		Reach me at sgusev@gmail.com
    </div>
</header>


            <h1 class="page-title post-title">
    <div class="page-title__text post-title__text">Hot takes: interesting papers from June</div>
    <div class="page-title__subtitle post-title__subtitle"></div>
</h1>

<div class="content">
    <h2 id="transcriptomeproteome">Transcriptome/Proteome</h2>

<p><strong><a href="http://www.nature.com/nature/journal/v534/n7608/full/nature18270.html">Defining the consequences of genetic variation on a proteome-wide scale, Chick et al. Nature</a></strong></p>

<p>“This study quantified both protein and transcript abundance in a genetically diverse population of mice, mapping their genetic architecture … We conclude that most local pQTL affected both protein and transcript abundance, consistent with a transcriptional mode of regulation. However, distant pQTL affected protein abundance independently of the transcript, consistent with a post- transcriptional mode of regulation.”</p>

<p><strong><a href="http://science.sciencemag.org/content/352/6293/1586">Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain, Lake et al. Science</a></strong></p>

<p>“Our results demonstrate that postmortem SNS [single-nucleus RNA sequencing] can identify expected and previously unidentified neuronal subtypes that provide insight into brain function through distinct profiles of activity- defining genes”</p>

<h2 id="epigenome">Epigenome</h2>

<p><strong><a href="http://www.nature.com/nrg/journal/vaop/ncurrent/abs/nrg.2016.59.html">The molecular hallmarks of epigenetic control, Allis et al. Nat Rev Genet</a></strong></p>

<p>“Here, we provide a personal perspective on the development of epigenetics, from its historical origins to what we define as ‘the modern era of epigenetic research’.”</p>

<p><strong><a href="http://journals.plos.org/plosgenetics/article?id=10.1371%2Fjournal.pgen.1006105">Epigenome-wide Association Studies and the Interpretation of Disease -Omics, Birney et al. PLOS Genet</a></strong></p>

<p>“As EWAS have generally been only rarely performed with concurrent genotyping of the same individuals or transcriptional studies of the same cells, we have no way of knowing whether the positive results of EWAS to date are testing the starting hypothesis that genuine epigenetic changes occur within a subset of cells in the population. Instead, the results may be due to residual meta-epigenomic effects of cell subtypes or attributable to untested influences of genomic or transcriptomic variability. This being the case, and with similar caveats affecting transcriptomic studies, no EWAS to date can be said to be fully interpretable … Analytically, insights into DNA sequence variants upon DNA methylation (methylation quantitative trait loci, mQTLs) for the cell type studied will allow approaches to be developed to account for this major influence upon the epigenome. One particular approach, two- step mendelian randomization, is being applied in prospective and case/control EWAS, build- ing on the non-modifiable nature of germline genetic variation to provide causal anchors within a causal inference setting.”</p>

<p><strong><a href="http://www.nature.com/nature/journal/vaop/ncurrent/full/nature18606.html">The landscape of accessible chromatin in mammalian preimplantation embryos, Wu et al. Nature</a></strong></p>

<p>“Here we provide a genome-wide survey of accessible chromatin in the mouse preimplantation embryos using ATAC-seq. A fundamental question in preimplantation development is to what extent gene expression is linked to epigenome reprogramming. Our data indicate that gene activation and establishment of open chromatin could occur, at least in part, through different pathways from those for epigenetic modification reprogramming.”</p>

<h2 id="heritabilitycorrelation">Heritability/correlation</h2>

<p><strong><a href="http://www.cell.com/ajhg/abstract/S0002-9297(16)30148-3">Contrasting the Genetic Architecture of 30 Complex Traits from Summary Association Data, Shi et al. AJHG</a></strong></p>

<p>“Here, we introduce methods that estimate the total trait variance explained by the typed variants at a single locus in the genome (local SNP heritability) from genome-wide association study (GWAS) summary data while accounting for linkage disequilibrium among variants. We applied our estimator to ultra-large-scale GWAS summary data of 30 common traits and diseases to gain insights into their local genetic architecture. First, we found that common SNPs have a high contribution to the heritability of all studied traits. Second, we identified traits for which the majority of the SNP heritability can be confined to a small percentage of the genome. Third, we identified GWAS risk loci where the entire locus explains significantly more variance in the trait than the GWAS reported variants. Finally, we identified loci that explain a significant amount of heritability across multiple traits.”</p>

<p><strong><a href="http://www.cell.com/ajhg/pdfExtended/S0002-9297(16)30135-5">Transethnic Genetic-Correlation Estimates from Summary Statistics, Brown et al. AJHG</a></strong></p>

<p>“We have developed transethnic genetic-effect and genetic- impact correlations and provided a method for estimating these quantities on the basis of only summary-level GWAS information and suitable reference panels … In all pheno- types analyzed, the genetic correlation was significantly different from both 0 and 1.”</p>

<p><strong><a href="http://www.cell.com/ajhg/pdf/S0002-9297(16)30132-X.pdf">Imputing Phenotypes for Genome-wide Association Studies, Hormozdiari et al. AJHG</a></strong></p>

<p>“We propose an approach called phenotype imputation that allows one to perform a GWAS on a phenotype that is difficult to collect. Our approach leverages the correlation structure between multiple phenotypes to impute the uncollected phenotype … Our method can impute the summary statistic of the target phenotype as a weighted linear combination of the summary statistics of related phenotypes.”</p>

<p><strong><a href="http://genome.cshlp.org/content/early/2016/06/14/gr.201996.115.abstract">Multikernel linear mixed models for complex phenotype prediction, Weissbrod et al. Gen Res</a></strong></p>

<p>“Here, we present multikernel linear mixed model (MKLMM), a flexible modeling framework that allows for both global and local high-order interactions modeled via RKHS, as well as modeling of a heterogeneous effect-size distribution … MKLMM-Adapt held a statistically significant advantage over AMB [adaptive multi-BLUP] in prediction of Crohn’s disease (CD), type 1 diabetes (T1D), and ulcerative colitis (UC), whereas AMB did not hold a statistically significant advantage over MKLMM-Adapt in any data set “</p>

<h2 id="computational-phasing">Computational phasing</h2>

<p><strong><a href="http://www.nature.com/ng/journal/v48/n7/full/ng.3571.html">Fast and accurate long-range phasing in a UK Biobank cohort, Loh et al. Nat Genet</a></strong></p>

<p>“The basic idea of our approach is to harness IBD from distant relatedness (up to ~12 generations from a common ancestor) that is pervasive within very large cohorts … we observed that Eagle analysis of all N ≈ 150,000 samples together completed three times faster than SHAPEIT2 10 × 15,000 analysis while achieving a 67% (1%) decrease in switch error rate: switch error rate of 0.30% (0.01%) for Eagle 1 × 150,000 versus 0.90% (0.06%) for SHAPEIT2 10 × 15,000 … corresponding to perfect phase in a majority of  10-Mb segments”</p>

<p><strong><a href="http://www.nature.com/ng/journal/v48/n7/full/ng.3583.html">Haplotype estimation for biobank-scale data sets, O’Connell et al. Nat Genet</a></strong></p>

<p>“SHAPEIT3 enhances SHAPEIT2 in two ways that enable it to deal with very large data sets, such as the UKB study. The first advance is based on the intuition that larger sample sizes are likely to result in increased local similarity between groups of haplotypes due to the higher probability of more recent shared ancestry … The second advance involves changes to the Markov chain Monte Carlo (MCMC) sampling routine that result in additional gains in speed. As sample size grows, it becomes more likely that two individuals will have a long stretch of sequence in common within a particular window … We removed the trio parents from the data set and phased the whole of chromosome 20 (16,265 genotyped sites) for the remaining 152,112 individuals. This run resulted in a median switch error rate of 0.4% and took 38.5 h using ten threads.</p>

<h2 id="popgen">Popgen</h2>

<p><strong><a href="http://www.nature.com/nature/journal/v534/n7606/full/nature17993.html">The genetic history of Ice Age Europe, Fu et al. Nature</a></strong></p>

<p>“Here we analyse genome-wide data from 51 Eurasians from ~45,000–7,000 years ago. Over this time, the proportion of Neanderthal DNA decreased from 3–6% to around 2%, consistent with natural selection against Neanderthal variants in modern humans. Whereas there is no evidence of the earliest modern humans in Europe contributing to the genetic composition of present-day Europeans, all individuals between ~37,000 and ~14,000 years ago descended from a single founder population which forms part of the ancestry of present-day Europeans.”</p>

</div>

<div class="about">
    <div class="about__devider">*****</div>
    <div class="about__text">
        Written by <strong>  Sasha Gusev </strong>
        on <strong>04 July 2016</strong>
    </div>
</div>


<div id="disqus_thread"></div>
<script type="text/javascript">
    /* * * CONFIGURATION VARIABLES * * */
    var disqus_shortname = 'sashagusev';

    /* * * DON'T EDIT BELOW THIS LINE * * */
    (function() {
        var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
        dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
        (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
</script>
<noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript" rel="nofollow">comments powered by Disqus.</a></noscript>



        </div>

        <script src="http://sashagusev.github.io/assets/vendor/highlight/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
        
    </body>
</html>